src/api.ts

```diff
@@ -240,31 +240,38 @@ export async function migrateList(
  * Check the filter condition and apply filter if required.
  *
  * @param {ConnectionReference[]} connections
- * @param {string} [filterConnectionId]
+ * @param {string} [filterConnectionIds]
  * @returns {ConnectionReference[]}
  */
 function filterConnectionsAsRequired(
   connections: ConnectionReference[],
-  filterConnectionId?: string
+  filterConnectionIds?: string
 ): ConnectionReference[] {
-  log(`Filter (--only=) ${filterConnectionId}`);
+  const trimmedFilterConnectionIds = filterConnectionIds?.split(',').map(id => id.trim());
+  const formattedFilterConnectionIds = trimmedFilterConnectionIds?.join(', ');
+
+  log(`Filter(s) (--only=) ${formattedFilterConnectionIds}`);
 
   // Apply no filter if the connection id is not provided.
-  if (!filterConnectionId) {
+  if (!filterConnectionIds) {
     log('Running for all connections.');
 
     return connections;
   }
 
-  const filteredList = connections.filter(connection => connection.id === filterConnectionId);
+  const filteredList = connections.filter(connection => trimmedFilterConnectionIds?.includes(connection.id));
 
   if (filteredList.length === 0) {
     const available = connections.map(({ id }) => id);
 
-    throw new Error(`No connections found for given id "${filterConnectionId}. Available ids are: ${available}`);
+    throw new Error(
```

Comment:
Do we need to handle case if at least one connection id is found and some of the ids aren't found? 

Example: For --only=id1,id2,id3, if id1 and id3 is present and id2 isn't present. Should we log some error for id3 or stop the process altogether? 

Also what if one does --only=id1,id1. Would it run sync-db twice for same connection. Let's remove duplicates just in case.

Reply:
Yes unique id would be good. Also will add a log in case we add entry for invalid connection id